### PR TITLE
User Rights on wikicanadawiki for T1064

### PR DESCRIPTION
**Pull request by LegoFan4000**

### CODE REVIEW

**Comment by Reception123** [comment]
As seen on line 3261, wgRevokePermissions doesn't work, therefore you need to do this in the overrides.

**Comment by LegoFan4000** [comment]
Done although it seems to work on login wiki and testwiki

**Comment by Reception123** [comment]
@LegoFan4000 As per what @JohnFLewis said in a previous commit, you need to add a "read" permission for the "banned" group as groups can't be assigned without permissions.

**Comment by LegoFan4000** [comment]
Adding the read permission would defeat the purpose of this PR as we are revoking the read permission

**Comment by Reception123** [comment]
@LegoFan4000 Yes, I did not realize that. Please assign another arbitrary permission then.

**Comment by JohnFLewis** [comment]
Repealing read is not the same as revoking read.

This first commit was correct to me.

**Comment by JohnFLewis** [comment]
Though you don't want to revoke read from *, you want to repeal that.

File: LocalSettings.php [hunk 1]
```diff
@@ -4105,11 +4113,6 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
 	$wgGroupPermissions['sysop']['abusefilter-modify-restricted'] = false;
 }
 
-if ( $wgDBname == 'wikicanadawiki' ) {
-	$wgGroupPermissions['*']['read'] = false;
```
Comment (JohnFLewis) [inline LocalSettings.php]: This was correct.

File: LocalSettings.php [hunk 2]
```diff
@@ -3265,6 +3265,14 @@
 				'editmyoptions' => true,
 			),
 		),
+		'wikicanadawiki' => array(
+ 			'*' => array(
+ 				'read' => true,
+ 			),
+ 			'banned' => array(
+ 				'read' => true,
```
Comment (JohnFLewis) [inline LocalSettings.php]: This is correct.

**Comment by JohnFLewis** [comment]
Though I dont have shell right now so i prefer @Reception123 merging right now.

**Pull request merged**